doc/source/dev/contributor/quickstart_docker.rst

```diff
@@ -0,0 +1,177 @@
+   from the container to any of the files in that directory are also
+   visible in the host, and vice-versa.
+
+#. You should now be in the container, with something like::
+
+      root@468e1b9564e4:/#
+
+   as a prompt.
+
+#. Navigate to the SciPy source directory, which is shared with the host OS.
+
+   ::
+
+      cd /home/scipy
+
+#. The container has both Python 3.6 and Python 3.7 available. To start
+   using/building SciPy, we need to install some dependencies::
+
+      pip3.7 install numpy cython pytest pybind11
+
+   If you want to work with Python 3.6 use the ``pip3.6`` command instead.
+
+#. Do an in-place build by entering::
+
+      python3.7 setup.py build_ext --inplace
+
+   This will compile the C,
+   C++, and Fortran code that comes with SciPy. ``setup.py`` is a
+   script in the root directory of SciPy, which is why you have to be
+   in the SciPy root directory to call it. ``build_ext`` is a command
+   defined in ``setup.py``, and ``--inplace`` is an option we’ll use to
+   ensure that the compiling happens in the SciPy directory you already
+   have rather than some other folder on your computer. If you want to
+   work with Python 3.6, replace ``python3.7`` with ``python3.6``.
+
+#. Test the build by entering::
+
+      python3.7 runtests.py -v
+
+   ``runtests.py`` is another script in the SciPy root directory. It runs a
+   suite of tests that make sure SciPy is working as it should, and ``-v``
+   activates the ``–verbose`` option to show all the test output.
+
+From here, you can start a Python console (e.g. enter ``python3.7``) or
+execute Python scripts from the command line (e.g.
+``python3.7 scriptname.py``). If you want to do this from any directory
+other than the SciPy root, you may want to add SciPy to the Python path::
+
+   export PYTHONPATH="/home/scipy:$PYTHONPATH"
+   source ~/.bashrc
+
+Otherwise, SciPy may not be visible to Python.
+
+You can make changes to files in the ``scipy`` directory in a text editor/IDE
+in your host OS, and those changes will be reflected
+within the container. Alternatively, you can use the ``vi``
+text editor within the container to make changes. No changes made
+within the container are retained when the container is exited; only
+changes made to files/folders within mounted volumes are kept.
+If you would like to contribute changes to the SciPy project, please see
+:ref:`development-workflow`.
+
+Finally, although Python and pip are pre-installed on the provided
+Docker image, you are welcome to to install a different
+Python distribution and package manager, such as Anaconda. In this case, you
+can adapt the instructions from :ref:`quickstart-ubuntu`, using the
+container as you would any other Linux terminal. You've already cloned
+SciPy on your computer, and git and all required compilers are already
+installed, so you can simply skip the corresponding steps
+(:ref:`quickstart-ubuntu-build` 2-7, 9, 10, and 13).
```

Comment:
fixed point numbers are going to go stale quickly.

Reply:
I know : / I debated. I wanted to make it simple, but I guess for the sake of maintenance it is better to leave them out.

Reply:
Let me unresolve until I make the change.